lumen/views/base.py

```diff
@@ -489,8 +493,19 @@ def panel(self) -> Viewable:
                 panel = panel.layout[0]
             else:
                 panel = panel._layout
-        if self.download:
-            return pn.Column(self.download, panel, sizing_mode='stretch_width')
+        if self.title:
+            title_pane = pn.pane.HTML(
+                f'<h3 align="center", style="margin-top: 0; margin-bottom: 0;">{self.title}</h3>',
+                sizing_mode="stretch_width"
+            )
+        if self.download and self.title:
+            download_pane = pn.panel(self.download)
+            download_pane.sizing_mode = "fixed"
+            return pn.Column(pn.Row(title_pane, download_pane), panel)
+        elif self.download:
+            return pn.Column(self.download, panel, sizing_mode="stretch_both")
+        elif self.title:
+            return pn.Column(title_pane, panel)
```

Comment:
I think this and the first branch need to inherit the sizing_mode from the panel, i.e.: 

```suggestion
            return pn.Column(title_pane, panel, sizing_mode=panel.sizing_mode)
```

But please test before applying.